plugins/c8y_remote_access_plugin/src/lib.rs

```diff
@@ -95,9 +93,12 @@ static SUCCESS_MESSAGE: &str = "CONNECTED";
 struct Unreachable<E: std::error::Error + 'static>(#[source] E, &'static str);
 
 async fn spawn_child(command: String, config_dir: &Utf8Path) -> miette::Result<()> {
-    let exec_path = get_executable_path(config_dir).await?;
+    let exec_path = std::env::current_exe()
+        .into_diagnostic()
+        .with_context(|| "Could not get current process executable")?;
 
     let mut command = tokio::process::Command::new(exec_path)
+        .args(["--config-dir", config_dir.as_str()])
```

Comment:
Yes I'm using tedge in a non  default location. Background is, I'm creating a homebrew formula to install thin-edge.io on MacOS and part of the homebrew convention is to install the package in a non-root location (e.g. not `/etc/tedge`).

I manually created the `c8y_RemoteAccessConnect` operation handler which now calls the binary using `--config-dir`, however since the command is now composed of a path and arguments, the existing logic treats it as one command thus failing because it does not exist. But anyway, since the c8y-remote-access-plugin is spawning itself, it should make sure it executes in the same context as it was launched.

Reply:
Technically one could create a symlink to `/etc/tedge` so that existing tooling works as expected, however this would required root access to create the symlink which is not possible in some situations, and there is not good reason to force people into requiring root access just for a folder location.